lib/livebook/runtime/definitions.ex

```diff
@@ -98,6 +98,13 @@ defmodule Livebook.Runtime.Definitions do
             %{name: "postgrex", dependency: %{dep: {:postgrex, "~> 0.16.3"}, config: []}}
           ]
         },
+        %{
+          name: "Snowflake",
+          packages: [
+            kino_db,
+            %{name: "adbc", dependency: %{dep: {:adbc, "~> 0.1.1"}, config: []}}
```

Comment:
We have to add explorer as well as a dependency. Even better, kino_explorer, so the results appear as a table.